### PR TITLE
Feature header

### PR DESCRIPTION
**Pull request by AvocadoCasualty**
mobile-menu-open
mobile-menu-closed
desktop-menu

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,475 @@
+{
```
Comment (thorbaert) [inline package-lock.json]: We're trying to not commit the package-lock.json

If you would be willing to just delete the file then commit it

File: public/src/app.scss [hunk 2]
```diff
@@ -3,17 +3,21 @@ $secondary: #606060;
 @import 'bootstrap/scss/bootstrap.scss';
 @import './_variables_&_mixins.scss';
 
+@import './shared/header';
```
Comment (thorbaert) [inline public/src/app.scss]: Doesn't this need to be `_header`?

**Review by thorbaert — CHANGES_REQUESTED** [review]

File: public/src/app.scss (hunk 2 above)
Comment (AvocadoCasualty) [reply public/src/app.scss]: Webstorm says it's unnecessary but I'll update it so it's more standard.

Comment (thorbaert) [reply public/src/app.scss]: Normally it isn't necessary but for some reason on ubuntu it breaks without it

File: package-lock.json (hunk 1 above)
Comment (AvocadoCasualty) [reply package-lock.json]: Done

**Review by thorbaert — APPROVED** [review]

**Pull request merged**